vision_agent/agent/vision_agent_planner_prompts_v2.py

```diff
@@ -651,22 +653,24 @@ def process_florence2_sam2_video_tracking(frames):
 """
 
 FINALIZE_PLAN = """
-**Role**: You are an expert AI model that can understand the user request and construct plans to accomplish it.
-
 **Task**: You are given a chain of thoughts, python executions and observations from a planning agent as it tries to construct a plan to solve a user request. Your task is to summarize the plan it found so that another programming agnet to write a program to accomplish the user request.
```

Comment:
nit: not related to the pr
```suggestion
**Task**: You are given a chain of thoughts, python executions and observations from a planning agent as it tries to construct a plan to solve a user request. Your task is to summarize the plan it found so that another programming agent to write a program to accomplish the user request.
```

Reply:
Good catch!